services/notification-service/app/main.py

```
@@ -0,0 +1,117 @@
+        await nc.subscribe(settings.nats_subject_order_completed, cb=_handle_nats_message)
+        logger.info(
+            "nats_subscription_started",
+            nats_url=sanitized_nats_url,
+            subject=settings.nats_subject_order_completed,
+        )
+    except Exception as exc:
+        if nc is not None and not nc.is_closed:
+            try:
+                await nc.close()
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)